packages/terra-toggle-button/src/ToggleButton.module.scss

```
@@ -7,6 +7,10 @@
     display: inline-block;
     font-size: 0.8em;
```

Quick fix - remove font-size from here and replace with missing line-height reset:
```suggestion
    line-height: 1rem;
```